src/job/jobHandler.ts

```diff
@@ -811,25 +808,52 @@ export abstract class JobHandler {
       }
 
       const url = `${previewWebhookURL}/${gatsbySiteId}`;
-      const response = await axios.post(
-        url,
-        {
-          jobId: this.currJob._id,
-        },
-        {
-          headers: { 'x-gatsby-cloud-data-source': 'gatsby-source-snooty-preview' },
-        }
-      );
-      await this._jobRepository.updateExecutionTime(this.currJob._id, { gatsbyCloudStartTime: new Date() });
-      await this.logger.save(this.currJob._id, `${'(POST Webhook Status)'.padEnd(15)}${response.status}`);
+      const response = await this.callExternalBuildHook(url, 'gatsbyCloudStartTime');
+      await this.logger.save(this.currJob._id, `${logPrefix.padEnd(15)} ${response.status}`);
     } catch (err) {
       await this.logger.save(
         this.currJob._id,
-        `${'(POST Webhook)'.padEnd(15)}Failed to POST to Gatsby Cloud webhook: ${err}`
+        `${logPrefix.padEnd(15)} Failed to POST to Gatsby Cloud webhook: ${err}`
       );
       throw err;
     }
   }
+
+  // Invokes Netlify Build Hook
+  protected async callNetlifyWebhook(): Promise<void> {
+    const githubUsername = this.currJob.user;
+    const logPrefix = '(EXPERIMENTAL - POST Netlify Webhook Status)';
+
+    try {
+      const url = await this._repoEntitlementsRepo.getNetlifyBuildHookByGithubUsername(githubUsername);
+      if (!url) {
+        const message = `User ${githubUsername} does not have a Netlify build hook.`;
+        this._logger.warn('Netlify Build Hook', message);
+        return;
+      }
+
+      const res = await this.callExternalBuildHook(url, 'netlifyStartTime');
+      await this.logger.save(this.currJob._id, `${logPrefix.padEnd(15)} ${res.status}`);
+    } catch (err) {
+      // Intentionally log and don't throw error since this is currently experimental and shouldn't block build progress
+      const errorMsg = `${logPrefix.padEnd(
+        15
+      )} Failed to POST to Netlify webhook. This should not affect completion of the build: ${err}`;
+      await this.logger.save(this.currJob._id, errorMsg);
+    }
+  }
+
+  protected async callExternalBuildHook(url: string, startTimeKey: string) {
```

Comment:
not-blocking at all: I don't remember if we already have a type or Partial type for the return value of this response. It could be more trouble than necessary at this point, but always nice to have a return type